Installation/Schema/brc/brc.besluitInformatieObject.schema.json

```diff
@@ -21,7 +21,7 @@
       "title": "Informatieobject",
       "description": "URL-referentie naar het INFORMATIEOBJECT (in de Documenten API) waarin (een deel van) het besluit beschreven is.",
       "type": "string",
-      "format": "uri",
+      "format": "url",
```

Comment:
Good to raise this: uri and url now have the same effect, however, for consistency sake url is better now.